sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blobs/tests/Azure.Messaging.EventHubs.CheckpointStore.Blobs.Tests.csproj

```diff
@@ -32,4 +32,12 @@
   <ItemGroup>
     <Compile Include="..\..\Azure.Messaging.EventHubs\tests\Shared\*.cs" Link="SharedSource\%(Filename)%(Extension)" />
   </ItemGroup>
+
+  <!-- TODO Replace with NuGet reference when preview 6 published -->
```

Comment:
We can't merge this way due to the problems that I mentioned with mixed references and build pipelines.    If we do decide the best approach is to `friend` the package, this can hang out until then; there's no urgency for this one.